docs/accessing-orcd/ssh-login.md

```diff
@@ -0,0 +1,103 @@
+
+    ####Other Notes about Windows  
+    Some programs may seem like valid terminals for accessing SuperCloud but are not ideal. Here are a few examples and why they are not recommended:
+
+    - <u>Windows Command Prompt (pre-Windows 10):</u> Does not natively support bash or SSH commands.
+    - <u>Windows PowerShell (pre-Windows 10):</u> Similar to Command Prompt, it does not natively support bash or SSH commands.
+    - <u>PuTTY:</u> A GUI-based program that is tricky to set up with SSH keys and requires a separate program for file transfers.
+
+## Logging in via SSH
+Once you have your terminal set up for your specific operating system, you can use SSH to access our HPC systems. Follow the commands below for your desired system.
+
+=== "Engaging"
+    The Engaging Cluster has 3 login nodes: 
+
+    - eofe4.mit.edu (Rocky 8)  
+    - eofe9.mit.edu (CentOS 7)  
+    - eofe10.mit.edu (Rocky 8)  
+    
+    To login via the command line, run the SSH command:
+    ```bash 
+    ssh [username]@[host]
+    ```
+    Replace `[username]` with your MIT Kerberos username and `[host]` with the desired login node name (e.g., ssh your_name@eofe9.mit.edu).
+
+    Connecting to eofe9 or eofe10 requires Two-Factor Authentication.
+
+    !!! Note
+        You will be prompted to enter your MIT Kerberos password if you have not set up SSH keys. To set them up, please refer to our [SSH Key Setup Page](https://orcd-docs.mit.edu/accessing-orcd/ssh-setup/).
+
+=== "Satori"
+    Satori has 2 login nodes:  
+
+    - Use `satori-login-001.mit.edu` for submitting training jobs and related activities.
+    - Use `satori-login-002.mit.edu` for transferring large files/datasets and compiling software.
+
+    To login via the command line, run the SSH command:
+    ```bash 
+    ssh [username]@[host]
+    ```
+    Replace `[username]` with your MIT Kerberos username and `[host]` with the desired login node name (e.g., ssh your_name@satori-login-001.mit.edu).
+
+    !!! Note
+        You will be prompted to enter your MIT Kerberos password if you have not set up SSH keys. To set them up, please refer to our [SSH Key Setup Page](https://orcd-docs.mit.edu/accessing-orcd/ssh-setup/).
+
+=== "SuperCloud"
+    Accessing the SuperCloud system through SSH requires that your public ssh-key has been added to the authorized_keys file in your SuperCloud account. Please note, if you have generated your key following the instructions at [supercloud.mit.edu](https://supercloud.mit.edu/requesting-account/#ssh-keys) or [orcd-docs](https://orcd-docs.mit.edu/accessing-orcd/ssh-setup/) and sent it to supercloud@mit.edu, then it should be in place for you. For further information on adding SSH keys to your account, refer to this [Page](https://supercloud.mit.edu/requesting-account/#adding-keys).
```

Comment:
They shouldn't send their ssh keys to supercloud@mit.edu anymore, you can use the following text in place of this paragraph:

Accessing the SuperCloud system through SSH requires that your public ssh-key has been added to the authorized_keys file in your SuperCloud account. Follow the instructions on [this page](https://mit-supercloud.github.io/supercloud-docs/requesting-account/#generating-ssh-keys) to create your keys and add them to your account.